### DISCUSSION TITLE
关于使用axios向saucenao发送请求，得到的是html格式的数据问题

### DISCUSSION THREAD

**Discussion by Minori-ty** [Tsuk1ko/CQ-picfinder-robot#160] [一般讨论]
您好，我自己用axios向saucenao发送请求，得到的是html格式的数据。我看了一下您的代码，看不出处理方式。请问您是怎么封装得到图片的json数据的？

**Comment by Tsuk1ko** [comment]
去看 saucenao 的 API 文档，有一个参数是输出形式
